esp/src/eclwatch/HPCCPlatformWidget.js

```diff
@@ -234,6 +235,22 @@ define([
                 topic.subscribe("hpcc/monitoring_component_update", function (topic) {
                     context.checkMonitoring(topic.status);
                 });
+                this.storage = new ESPUtil.LocalStorage();
+                this.storage.start();
+                this.storage.on("storageUpdate", function(event) {
+                    context._onUpdateFromStorage(event)
+                });
+            },
+
+            _onUpdateFromStorage: function (value){
+                var context = this;
+                if (value.event.newValue === "logged_out") {
+                    window.location.reload();
+                } else if (value.event.newValue === "Locked") {
+                    context._onShowLock();
+                } else if (value.event.newValue === "Unlocked" || value.event.oldValue === "Locked") {
+                    context._onHideLock();
+                }
```

Comment:
Minor suggestion - it might be worth while declaring the known events in ESPUtil:
```
LOGGED_OUT = "logged_out";
LOCKED = "locked";
...
```

Reply:
You mean within HPCCPlatformWidget?  I want to keep ESPUtil as non session management specific as possible. So, that others can use it without having to add their own variables within ESPUtil. Ideas?